franka:
  urdf:
    package: franka_panda_description
    path: robots/panda_arm.urdf.xacro
  position: [ 0, 0, 0 ]
  rpy: [ 0, 0, 0 ]
  fixed_base: True
  use_inertia_from_file: True
  #  max_effort: 100.0 # the max force to apply to the robot joints in vel mode
  plugins:
    - module: pybullet_ros2.plugins.torque_control
      class: TorqueControl
    - module: pybullet_ros2.plugins.joint_state_publisher
      class: JointStatePublisher
